lib/src/execute.rs

```diff
@@ -281,9 +276,8 @@ impl ExecuteCtx {
             .expect("`memory` is exported")
             .size(&store);
 
-        event!(
-            Level::INFO,
-            "Request completed, using {} bytes of wasm heap",
+        info!(
+            "request completed using {} bytes of WebAssembly heap",
```

Comment:
Maybe a future improvement and out of scope of this PR, however, it would be nice if the `n bytes` could be a more human-friendly representation I.e. 1 Kb, 1Mb, 1Gb  etc. Such as [bytesize](https://docs.rs/bytesize/1.0.1/bytesize/).

Reply:
Sounds great, I'll try to sneak that one in as well!

Reply:
Alright, I added a commit to make the heap size human-friendly, thank you!